src/main/node/config-manager.ts

```diff
@@ -1,144 +1,116 @@
 import { isDirectory, isPublication } from './file-manager';
+import { Collaborator } from '../../shared/redux/slices/publicationsSlice';
+import { configFileName } from './config-util';
+import { Configuration } from './model/Configuration';
 
 const fs = require('fs');
 
-// TODO: This should be switched over to definition form #93 -> https://github.com/vnLab-Lodz/PubLab/pull/93
-class Collaborator {
-  username: string;
-
-  role: string;
+export function createConfigFile(path: string, configuration: Configuration) {
+  validatePath(
+    !isDirectory(path),
+    'Error while creating a config file: directory is not found'
+  );
+  if (
+    !configuration.publicationName ||
+    !configuration.packageManager ||
+    !configuration.collaborators
+  ) {
+    throw new Error('Name, package manager and collaborators are required');
```

Comment:
I think that only the `publicationName` should not be empty as it's user input but other fields will be handled with defaults so no need to check them,